VideoLocker/src/main/java/org/edx/mobile/module/notification/NotificationFlyinService.java

```diff
@@ -0,0 +1,153 @@
+package org.edx.mobile.module.notification;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.PixelFormat;
+import android.media.RingtoneManager;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.edx.mobile.R;
+import org.edx.mobile.util.ResourceUtil;
+import org.edx.mobile.util.UiUtil;
+import org.edx.mobile.view.Router;
+
+/**
+ * before Android 5, flyin notification window is not supported.
+ */
+public class NotificationFlyinService extends Service {
+
+    public static  int ID_NOTIFICATION = 2018;
```

Comment:
@aleffert  I will remove it.  Right now we dont use notification id.  as in Parse's android sdk, 
we overwrite 
protected android.app.Notification getNotification(android.content.Context context, android.content.Intent intent) {}
to create a custom notification if needed,  but notification id is only used when the notification is fired by NotificationManager.  we have no access to that using current approach.